.claude/skills/career-ops/SKILL.md

```
@@ -76,13 +78,18 @@ Read `modes/_shared.md` + `modes/{mode}.md`
 
 Applies to: `auto-pipeline`, `oferta`, `ofertas`, `pdf`, `contacto`, `apply`, `pipeline`, `scan`, `batch`
 
+### Modes that require only their mode file (+ `_shared.md` loaded via mode file itself):
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)